Convergence/IO/EPICS/CA/ChannelHandle.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace Convergence.IO.EPICS.CA

```

Comment:
Why you need a separate ChannelHandle for just CS?

Can't this be generic? where a single ChannelHandle should cover any type of IO?

Reply:
I missed this